src/app/charts/core/files/secrets/tpl/arkcase-server.yaml

```diff
@@ -12,6 +13,10 @@ pdftron:
   {{- end }}
 {{- end }}
 
+{{- with $aggrid.data }}
+aggrid.license: {{ . | b64dec | quote }}
```

Comment:
If there is no aggrid block, will this aggrid.license just be empty?  Or will it cause an error?

Reply:
it will not create `aggrid.license` key in the file at all